src/dodal/devices/oav/pin_image_recognition/manual_test.py

```diff
@@ -0,0 +1,33 @@
+"""
+Note: this file exists exclusively to make it easier to manually run
+image recognition on a beamline without setting up all of the
+bluesky infrastructure.
+
+It is otherwise unused.
```

Comment:
Nit: Can we use a `# pragma: no cover` here to placate coverage?

Reply:
@Tom-Willemsen This still didn't work in CI, not sure why. Can you either remove or work out why? Don't mind too much either way.